tests/integration/src/setup.rs

```diff
@@ -23,17 +23,95 @@ pub use frame_support::{
     traits::{OnFinalize, OnIdle, OnInitialize},
     weights::Weight,
 };
-pub use sp_core::H160;
+pub use pallet_evm::AddressMapping;
+pub use sp_core::{H160, H256, U256};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements for `pallet_evm::AddressMapping`, `sp_core::{H160, H256, U256}`, and `sp_runtime::MultiAddress` are not necessary as they are already imported in the previous section. Please remove lines 26-27.



<!-- This is an auto-generated comment by OSS CodeRabbit -->